### ISSUE TITLE
Retire BlackMarble?

### ISSUE THREAD

**Issue by chahank** [CLIMADA-project/climada_petals#77]
As far as I understand, the functionalities from the module BlackMarble are all also implemented in the LitPop module. Should we then consider retiring BlackMarble?

**Comment by peanutfun** [comment]
Why was I assigned? I have no idea 😅 From a quick glance, `BlackMarble` uses functions from `climada.entity.exposures.litpop`. So it seems like it adds some functionality on top?

@aleeciu @sameberenz @gabrielaznar @tovogt You are involved in the history of the source files. We would highly appreciate some insight into what `BlackMarble` does that is currently *not* provided by `LitPop`.

**Comment by aleeciu** [comment]
As far as I understand, `BlackMarble` is a `LitPop` exposure which makes no use of pop data. If so, I am in favour of removing `BlackMarble` and:
- adding a line in the `LitPop` docstring on how one can still generate BlackMarble 
OR
- keeping a simple function called `BlackMarble` which simply calls LitPop with the right params

**Comment by chahank** [comment]
> Why was I assigned? I have no idea 😅

My logic was that you are the last that has had a detailed look at the LitPop code, so you might now best :D

**Comment by gabrielaznar** [comment]
That's right; BlackMarble only uses nightlight and not population. You can therefore get BlackMarble from LitPop as well. There were though some small differences using the nightlights in BlackMarble and LitPop (e.g. coupling the images) but they shouldn't change much the results. 

I think the only big difference now is that BlackMarble supports using the NOAA nightlights which are available yearly from 1992 till 2013. NASA nightlights are only available for 2012 and 2016 but at higher resolution than NOAAs (~500m vs ~1km).

**Comment by aleeciu** [comment]
this a good point - I guess having access to past data might be handy in some cases

**Comment by peanutfun** [comment]
The utility functions for downloading NASA and NOAA nightlights, which are used by `BlackMarble`, are actually part of the Core. However, they are never used there. They seem to have a strong overlap with the other functions implemented in Core, see [`nightlight.py`](https://github.com/CLIMADA-project/climada_python/blob/47b80ad946fecc48025455802e540b99256cc758/climada/entity/exposures/litpop/nightlight.py). If we consolidate them, we can add the functionality of downloading NOAA files to the Core nightlight implementation. This would then allow to remove `BlackMarble`.
